example/colors/app.js

```diff
@@ -12,9 +12,9 @@ var schema = {
   name: String,
 };
 
-var Color = app.model('color', schema);
-
+var Color = app.registry.createModel('color', schema);
 app.dataSource('db', { adapter: 'memory' }).attach(Color);
+app.model(Color);
```

Comment:
While you are changing this example, please rework it to the following:

```js
app.dataSource('db', { connector: 'memory' });

var Color = app.registry.createModel('color', schema);
app.model(Color, { dataSource: 'db' });
```
var Color = app.